examples/kubernetes-guestbook/.gitignore

```diff
@@ -0,0 +1,8 @@
+### Scala an JVM
+*.class
+*.log
+../components/.bsp
+components/.scala-build
```

Comment:
```suggestion
.bsp
.scala-build
```

Reply:
gitignore is recursive, so this should work for both simple and components directories